code/go/0chain.net/smartcontract/benchmark/main/config/benchmark.yaml

```diff
@@ -1,12 +1,12 @@
 simulation:
-  num_clients: 210
-  num_active_clients: 210
-  num_miners: 100
-  num_active_miners: 100
-  nun_sharders: 20
-  nun_active_sharders: 20
-  num_allocations: 300
-  num_blobbers: 100
+  num_clients: 21
```

Comment:
rollback changes please

Reply:
These changes are there for testing. It saves about 5-10 minutes each time I run the code. I'll change back when I have finished testing.